apps/test/unit/lib/kits/maker/sampleSerialPorts.js

```diff
@@ -25,6 +25,27 @@ export const CIRCUIT_PLAYGROUND_PORTS = [
     productId: "0x8011"
   }
 ];
+export const CIRCUIT_PLAYGROUND_EXPRESS_PORTS = [
+  {
+    comName: "COM5",
```

Comment:
I manufactured the COM5 entry here, just figured it's worth covering a likely case on Windows.